server/data/mappers/actionPlanMapper.ts

```diff
@@ -23,15 +22,10 @@ const toGoals = (response: GetGoalsResponse, prisonNamesById: Map<string, string
  * Sets a goal sequence by creation date (oldest = 1) and returns them in creation order (newest first)
  */
 function toOrderedGoals(goals: GoalResponse[], prisonNamesById: Map<string, string>): Goal[] {
-  return [...goals]
-    .sort((left: GoalResponse, right: GoalResponse) =>
-      dateComparator(toDate(left.createdAt), toDate(right.createdAt), 'ASC'),
-    )
-    .map((goal, index) => toGoal(goal, index + 1, prisonNamesById))
-    .reverse()
+  return [...goals].map(goal => toGoal(goal, prisonNamesById))
```

Comment:
Is the function name and its corresponding doc now accurate? It's less about ordering/sequencing them, and now just a mapping operation? 
Given it's a simple mapping, could it be inlined to where `toOrderedGoals` is called (unless it is called from several places?)

Reply:
Good spot, inlined.